src/shared-components/optionButton/index.tsx

```diff
@@ -4,17 +4,15 @@ import PropTypes from 'prop-types';
 import Style from './style';
 import { CheckmarkIcon } from '../../icons';
 import { isDefined } from '../../utils/isDefined';
+import { ThemeType } from '../../constants';
 
 export interface OptionButtonProps {
-  borderRadius?: string;
+  borderRadius?: valueof<ThemeType['BORDER_RADIUS']>;
```

Comment:
I considered removing this prop entirely--it is not currently used--but decided against it (for now). This will make sure we only pass in `theme`-consistent values. However, it does not check if it's appropriate for the theme, e.g. you can forcibly pass in `'16px'` for Agency and it won't raise any errors.